src/popup/pages/Retip.vue

```diff
@@ -76,6 +76,7 @@ import {
   useMaxAmount,
   useGetter,
   useState,
+  IFormModel, IMaxAmount,
```

Comment:
All entries above are split into new lines, so I suggest to do the same with this line.